samples/net/zperf/src/zperf_shell.c

```diff
@@ -218,6 +218,46 @@ int zperf_get_ipv4_addr(const struct shell *shell, char *host,
 	return 0;
 }
 
+struct in_addr *zperf_get_default_if_in4_addr(void)
+{
+	struct in_addr *in4_addr = NULL;
+#if CONFIG_NET_IPV4
+	struct net_if *iface;
+	struct net_if_ipv4 *ipv4;
+
+	iface = net_if_get_default();
+	ipv4 = iface->config.ip.ipv4;
+	for (int i = 0; i < NET_IF_MAX_IPV4_ADDR; i++) {
+		if (ipv4->unicast[i].is_used &&
+		     ipv4->unicast[i].addr_state == NET_ADDR_PREFERRED &&
```

Comment:
Something wrong with indentation in this line

Reply:
Another PR.